environment/env.go

```diff
@@ -26,3 +26,5 @@ var RaplPassword = "RAPL_PSSWD"
 
 // Location of the script that sets the powercap value for a host.
 var RaplThrottleScriptLocation = "RAPL_PKG_THROTTLE_SCRIPT_LOCATION"
+
+var LogConfigYaml = "logConfig.yaml"
```

Comment:
This file is intended to only store environment variables.

Reply:
Removed it from here. Added a flag in scheduler.go